proto/viam/app/packages/v1/packages.proto

```diff
@@ -93,6 +94,7 @@ message InternalPackage {
   google.protobuf.Struct metadata = 6 [(tagger.v1.tags) = "bson:\"metadata\" json:\"metadata\""];
   string blob_path = 7 [(tagger.v1.tags) = "bson:\"blob_path\" json:\"blob_path\""];
   google.protobuf.Timestamp created_on = 8 [(tagger.v1.tags) = "bson:\"created_on\" json:\"created_on\""];
+  string checksum = 9 [(tagger.v1.tags) = "bson:\"checksum\" json:\"checksum\""];
```

Comment:
[nit] For the future we shouldn't add `InternalPackage` to our API proto if it never is surfaced in our APIs. Instead we should define golang struct DB models in the app that are used for mongo vs our APIs. The fleet team is migrating all our usage to that model now.

Reply:
We were considering what the best practice was was initially defining models that are used for Mongo, so this is great to know. Created https://viam.atlassian.net/browse/DATA-1057 to track in our backlog.